languages/en_US.po

```diff
@@ -674,644 +837,780 @@ msgstr ""
 
-#: admin_functions.php:66
+#: admin_functions.php:68
 msgid "我是 Axton Yao，这个主题由我开发。我的网站是"
 msgstr "I am Axton Yao, the developer of this theme. My personal website is"
 
-#: admin_functions.php:67
+#: admin_functions.php:68 admin_functions.php:69 admin_functions.php:71
+msgid "。"
+msgstr "."
+
+#: admin_functions.php:69
 msgid "对主题有任何疑问，建议先查阅 "
-msgstr "Should you have any trouble, please refer to "
+msgstr "Should you have any questions, please refer to "
 
-#: admin_functions.php:67
+#: admin_functions.php:69
 msgid "主题文档"
 msgstr "the theme documentation"
 
-#: admin_functions.php:68
+#: admin_functions.php:70
 msgid "这个项目的 Github 地址是 "
-msgstr "The GitHub address for this project is "
+msgstr "This project is open sourced on GitHub at "
 
-#: admin_functions.php:68
+#: admin_functions.php:70
 msgid "。如果你有兴趣，欢迎为这个项目做出贡献。同时，求 Star。"
 msgstr ". Pull requests and stars are welcomed."
 
-#: admin_functions.php:69
+#: admin_functions.php:71
 msgid ""
 "这个主题的诞生离不开 MDUI，这是一个优秀的前端框架项目，你可以在他们的官方网站"
 "上了解更多："
 msgstr ""
 "This theme is inspired by MDUI, a promising frontend framework. You can "
 "learn more on their website:"
 
-#: admin_functions.php:71
+#: admin_functions.php:73
 msgid "感谢以下译者帮助翻译 MDx："
-msgstr "Thanks to following translators:"
+msgstr "Special thanks to the following translators:"
 
-#: admin_functions.php:78
+#: admin_functions.php:80
 msgid "当前版本 v"
 msgstr "Current Version v"
 
-#: admin_functions.php:79
+#: admin_functions.php:81
 msgid "构建版本 v"
 msgstr "Build Version v"
 
-#: admin_functions.php:80
+#: admin_functions.php:82
 msgid "最新版本 v"
 msgstr "Latest Release v"
 
-#: admin_functions.php:82
+#: admin_functions.php:84
 msgid "这款主题献给 Demi Zhou"
-msgstr "This theme be dedicated to Demi Zhou"
+msgstr "This theme is dedicated to Demi Zhou"
 
-#: admin_functions.php:117
+#: admin_functions.php:119
 msgid ""
 "MDx 主题现已激活，你可以前往<a href=\"admin.php?page=mdx_styles\">主题设置页"
 "面</a>对主题进行个性化定制。"
 msgstr ""
-"MDx theme is now activated, you can go to <a href=\"admin.php?page=mdx_styles"
-"\">theme settings</a> to customize it."
+"The MDx theme is now activated, you can go to <a href=\"admin.php?"
+"page=mdx_styles\">theme settings</a> to customize it."
 
-#: admin_functions.php:122
+#: admin_functions.php:124
 msgid ""
 "由于你的 PHP 配置问题，MDx 未能成功获取密钥，但这不影响你正常使用 MDx 主题。"
 msgstr ""
 "Due to your PHP configuration, MDx cannot acquire the key. Don't worry, MDx "
 "can still function normally."
 
-#: admin_functions.php:127
+#: admin_functions.php:129
 msgid ""
 "你似乎没有启用 <code>mbstring</code> PHP 拓展，这会导致 MDx 无法正常运行。请"
 "参阅<a href=\"https://doc.flyhigher.top/mdx/zh-CN/tech/php-errors/"
 "#mb_strimwidth\">文档</a>来修复此错误。"
 msgstr ""
 "PHP <code>mbstring</code> extension not enabled. MDx may not function "
 "normally. Please refer to <a href=\"https://doc.flyhigher.top/mdx/en-US/tech/"
-"php-errors/#mb_strimwidth\">documents</a> to fix this error."
+"php-errors/#mb_strimwidth\">the documentation</a> to fix this error."
+
```

Comment:
This doc hasn't been translated to `en-US` yet

Reply:
Since I'm still testing the theme, the doc has not been updated yet

Reply:
The English doc is not fully translated